proto/spike-service/spike.proto

```diff
@@ -33,4 +34,25 @@ message ValidateTokenResponse {
     bool valid = 1;
     repeated string scopes = 2;
     string message = 3;
+    string alias = 4;
+    User user = 5;
+}
+
+message ValidateAuthCodeTokenRequest {
```

Comment:
Is 'validate' the right term here? You're not just validating. perhaps 'extraction'?